apis/runtime/v1alpha1/api.proto

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// To regenerate api.pb.go run scripts/generate_runtime.sh
+syntax = "proto3";
+
+package runtime.v1alpha1;
+option go_package = "github.com/koordinator-sh/koordinator/apis/runtime/v1alpha1";
+
+
+// PodSandboxMetadata holds all necessary information for sandbox name.
```

Comment:
PodSandboxMetadata holds all necessary information for sandbox.

Reply:
fixed

Reply:
> PodSandboxMetadata holds all necessary information for sandbox.
fixed